### PR TITLE
Create 2023-Q2-Hyperledger-Cello.md

### PR DESCRIPTION
**Pull request by fengyangsy**
Submit Cello's Q2 2023 Summary.

### CODE REVIEW

**Review by arsulegai — APPROVED** [review]

**Review by petermetz — APPROVED** [review]
LGTM

FYI: I tried to execute the quick-start steps and then this happened :-)
https://github.com/hyperledger/cello/issues/528

File: project-reports/2023/2023-Q2-Hyperledger-Cello.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Required Information
+- [x] Have you switched from master to main in all your repos? 
+- [x] Have you implemented the [Common Repository Structure](../guidelines/repository-structure.md) in all your repos? 
+- [x] Has your project implemented these inclusive language changes listed below to your repo? You can optionally [use the DCI Lint tool](https://github.com/petermetz/gh-action-dci-lint#usage) to make this a recurring action on your repo.
+    - master → main
+    - slave → replicas
+    - blacklist → denylist
+    - whitelist → allowlist
+- [x] Have you added an [Inclusive Language Statement](https://wiki.hyperledger.org/display/TSC/Inclusive+Language+Example) to your project's documentation and/or Wiki pages?
+
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q2-Hyperledger-Cello.md]:
```suggestion
```

This is no longer part of the template.

**Review by tkuhrt — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Cello.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Cello
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+V1.0.0 version has been released.
```
Comment (lehors) [inline project-reports/2023/2023-Q2-Hyperledger-Cello.md:10]: It doesn't seem like this has happened yet and it's contradicted by the following statement about working towards that very release. Please, clarify.

**Review by lehors — CHANGES_REQUESTED** [review]
Looks good overall but there is a discrepancy in the report that needs to be addressed before it gets approved.
Thanks.

File: project-reports/2023/2023-Q2-Hyperledger-Cello.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Testing and fixing bugs;
+* Finishing the node operations;
+* Rewrite the installation scripts;
+* Improve documentation.
+
+# Current Plans
+
+Next steps include:
+
+* Finish the v1.1.0 GA release in 2023, which will include more operations of chaincode.
```
Comment (VRamakrishna) [inline project-reports/2023/2023-Q2-Hyperledger-Cello.md]: "operations of chaincode" sounds vague. Can you be a bit more descriptive?

**Review by VRamakrishna — CHANGES_REQUESTED** [review]
Please address the comments.

**Review by denyeart — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Cello.md (hunk 2 above)
Comment (lehors) [reply project-reports/2023/2023-Q2-Hyperledger-Cello.md:10]: With help from @tkuhrt and @ryjones I can now see that 1.0 really is out. So it's probably the next section that needs fixing. Also not that the CHANGELOG.md file is missing info on 1.0.

File: project-reports/2023/2023-Q2-Hyperledger-Cello.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+None
+
+# Releases
+
+* v1.0.0-beta1 is done in November, 2022.
+* v1.0.0-GA has been released in May, 2023.
+
+# Overall Activity in the Past Quarter
+
+The team is working mainly on the remaining work for the 1.0.0-GA release.
```
Comment (lehors) [inline project-reports/2023/2023-Q2-Hyperledger-Cello.md]: Given that 1.0 is out, it seems like this refers to what happened and should read:
```suggestion
The team was working mainly on the remaining work for the 1.0.0-GA release.
```

**Review by jimthematrix — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Cello.md (hunk 3 above)
Comment (fengyangsy) [reply project-reports/2023/2023-Q2-Hyperledger-Cello.md]: Detailed content has been submitted, please review, thanks.

File: project-reports/2023/2023-Q2-Hyperledger-Cello.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+# Current Plans
+
+Next steps include:
+
+* Finish the v1.1.0 GA release in 2023, which will include more operations of chaincode, for example upload, install, approve, and commit.
+
+# Maintainer Diversity
+
+Current there are 3 maintainers, and those active developers who contribute to Cello continuously (3 month) may be nominated as new maintainers.
```
Comment (arsulegai) [inline project-reports/2023/2023-Q2-Hyperledger-Cello.md:42]: I have already approved the report, but a question to the maintainers. Have you considered collaboration with other projects within Hyperledger Foundation which focuses on deployments and operations? Based on my understanding there is possible collaboration opportunity with the Hyperledger Bevel team for the helm charts/operators.

**Review by lehors — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Pull request merged**